app/controllers/courses_controller.rb

```diff
@@ -69,6 +77,11 @@ def destroy
     end
   end
 
+  def lecture_for_current_course(lecture, course)
```

Comment:
This is unused. it fell through the cracks when I was cleaning up. I'll get rid of it.